lang/python/python-awesomeversion/Makefile

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2021 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-awesomeversion
+PKG_VERSION:=21.8.1
+PKG_RELEASE:=1
+
+PYPI_NAME:=awesomeversion
+PKG_HASH:=a72f3fff12df257e30f5e25a8550fc4da62e7591d5fdb70714f5c96827c37645
+
+PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE.md
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-awesomeversion
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=AwesomeVersion
+  URL:=https://github.com/ludeeus/awesomeversion
+  DEPENDS:= \
+      +python3-light \
+      +python3-logging
+endef
+
+define Package/python3-awesomeversion/description
+  One version package to rule them all, One version package to find them, One version package to bring them all, and in the darkness bind them.
+endef
+
+$(eval $(call Py3Package,python3-awesomeversion))
+$(eval $(call BuildPackage,python3-awesomeversion))
+$(eval $(call BuildPackage,python3-awesomeversion))
```

Comment:
duplicate `$(eval $(call BuildPackage,python3-awesomeversion))`
i think this was supposed to be `python3-awesomeversion-src`

Reply:
I wonder from which package I took it. :shrug: 
Thank you for noticing! Fixed.